default.nix

```diff
@@ -1,5 +1,5 @@
-{ rev    ? "058f304d41966672be66efd978210b9ec8c81687" # release-21.11
-, sha256 ? "1l0x61jpiw2q78p7bgjk93slgqd3nbbfp0mxlivf6jhsmiymzpzj"
+{ rev    ? "6047d0269b0006756103db57bd5e47b8c4b6381b" # release-21.11
```

Comment:
which release is it? still 21-11?

Reply:
Yes, the same one as in `ic-hs`.

Reply:
in ic-hs it's already 22.11

Reply:
Oh sorry missed the 21. Will adapt the comment. Thanks.